### ISSUE TITLE
Which model is better for extracting human voice without harmonies?

### ISSUE THREAD

**Issue by imagine725** [Anjok07/ultimatevocalremovergui#555]
i want a pure vocal！！！

**Comment by RavenV-tiff** [comment]
My work flow is to separate the instrumental and vocal using ensemble avg/avg then take the vocal track of that and run the MDX karaoke 1/2 models. The vocal separation should leave the main vocals in theory. It can have artifacts since there really isn't a lot of models that separate backing vocals.

**Comment by RavenV-tiff** [comment]
I use UVR-MDX inst HQ 2 + inst main + inst_full_292 + 9482 + htdemucs. All the MDX use htdemucs as a secondary model. I will say recently I've moved to using MVSEP (https://github.com/ZFTurbo/MVSEP-MDX23-music-separation-model) as it produces the best quality separation I've found so far. Just speculation but that model and others from the MDX23 challenge will eventually be added to UVR

**Comment by RavenV-tiff** [comment]
Yeah, I read about it shortly after posting. Official agreement of the challenge, for any prizes, were to have the model open-source after it was over, so hopefully those that can't/won't were disqualified from that. Either way, new and improved methods are always on the horizon.
